webprofile/10/_index.md

```diff
@@ -1,26 +1,28 @@
 ---
-title: "Jakarta EE Web Profile 10 (under development)"
-date: 2021-08-11
+title: "Jakarta EE Web Profile 10"
+date: 2022-08-29
 summary: "Release of the Jakarta EE 10 Web Profile"
 ---
 The Jakarta EE Web Profile defines a profile of the Jakarta EE Platform specifically targeted at web applications.
 
+# Artifacts
+
 * [Jakarta Web Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/web-profile-10)
   * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
-* [Jakarta Web Profile 10 Specification Document](./jakarta-webprofile-spec-10.pdf) (PDF)
-* [Jakarta Web Profile 10 Specification Document](./jakarta-webprofile-spec-10.html) (HTML)
+* [Jakarta Web Profile 10 Specification Document](./jakarta-webprofile-spec-10.0.pdf) (PDF)
+* [Jakarta Web Profile 10 Specification Document](./jakarta-webprofile-spec-10.0.html) (HTML)
 * [Jakarta Web Profile 10 Javadoc](./apidocs)
 * [Jakarta Web Profile 10 TCK](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/platform/10/jakarta-jakartaeetck-10.0.0.zip.sha256),[pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
 * Maven coordinates
   * [jakarta.platform:jakarta.webprofile-api:jar:10.0.0](https://search.maven.org/artifact/jakarta.platform/jakarta.jakartaee-web-api/10.0.0/jar)
 * Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
-  * [TBD]()
+  * [Eclipse GlassFish Web Profile](https://download.eclipse.org/ee4j/glassfish/web-7.0.0-M8.zip)
```

Comment:
Shouldn't the version be specifiec?

"Eclipse GlassFish 7.0.0-M8 Web Profile"

Reply:
Updated to Eclipse GlassFish Web Profile 7.0.0-M8